api-report/container-runtime.api.md

```diff
@@ -661,12 +669,12 @@ export class Summarizer extends EventEmitter implements ISummarizer {
     }
 
 // @public (undocumented)
-export type SummarizeResultPart<T> = {
+export type SummarizeResultPart<T, Y = undefined> = {
```

Comment:
When there are multiple type parameters, I usually switch to words with a "T" prefix, like `<TAckResult, TNackResult>`.